tests/unit/inference/test_inference.py

```diff
@@ -383,6 +383,35 @@ def test(
         assert assert_fn(bs_output, ds_output)
 
 
+@pytest.mark.seq_inference
+@pytest.mark.parametrize("model_w_task", [("gpt2", "text-generation")], ids=["gpt2"])
+class TestLowCpuMemUsage(DistributedTest):
+    world_size = 1
+
+    def test(
+        self,
+        model_w_task,
+        dtype,
+        query,
+        inf_kwargs,
+        assert_fn,
+    ):
+        model, task = model_w_task
+        local_rank = int(os.getenv("LOCAL_RANK", "0"))
```

Comment:
```suggestion
        query,
        inf_kwargs,
        assert_fn,
    ):
        model, task = model_w_task
        dtype = torch.float16
        local_rank = int(os.getenv("LOCAL_RANK", "0"))
```

We don't need to test this over multiple dtypes